src/pages/steps/AdditionalInformationStep/MediaStep.tsx

```diff
@@ -193,9 +193,13 @@ const MediaStep = ({
       return enrichedVideo;
     });
 
-    const data = await Promise.all(convertAllVideoUrlsPromises);
+    const data = await Promise.allSettled(convertAllVideoUrlsPromises);
 
-    setVideos(data);
+    const successVideos = data
+      .filter((res) => res.status === 'fulfilled')
+      .map((res: PromiseFulfilledResult<VideoEnriched>) => res.value);
```

Comment:
```suggestion
  const isFulfilledResult = <T>(res: PromiseSettledResult<T>): res is PromiseFulfilledResult<T> => {
    return res.status === 'fulfilled'
  }

  const successVideos = data.filter(isFulfilledResult).map(res => res.value);
```

You can use a typescript narrowing function like here so you don't need the type annotation. I would put the `isFulfilledResult` in a different file so we can reuse it for other cases.

I also implemented this on UIV https://github.com/cultuurnet/uitbe-frontend/blob/main/packages/app/utilities/Promise.ts 😄